src/main/scala/ai/privado/utility/Utilities.scala

```diff
@@ -371,4 +373,24 @@ object Utilities {
     builder.addNode(node)
     builder.addEdge(node, fileNode, EdgeTypes.SOURCE_FILE)
   }
+
+  def semanticExporter(sourceRepoLocation: String, headers: List[String], semantics: List[Seq[String]]): Unit = {
```

Comment:
Instead of taking 2 lists as input, better take a map as input with key being the Header and value Seq/List of String